tests/unit/test_reconcile.py

```diff
@@ -92,6 +92,40 @@ def test__local_and_server_error(
         )
 
 
+def test__get_server_content_missing_link():
+    """
+    arrange: given path info with a file and table row with no changes and discourse client that
```

Comment:
The arrange description is exactly the same for this test and the next one while their behavior differs in terms of excpected exception
The main difference is the presence of the link in the navlink that changes the behavior of the method. 

I think the arrange description must be adapted

Reply:
Yes, this was not updated as much as needed, should be fixed shortly